src/UI/styles/globals.css

```diff
@@ -32,3 +32,7 @@ input[type='range']::-webkit-slider-thumb {
   cursor: pointer;
   box-shadow: 0 0 2px 0 #555;
 }
+
```

Comment:
css should be applied at the local level of the component rather than global css

Reply:
although I'm now wondering if this is just a leftover from making the layout work correctly as I'm not sure the scroll is red at the moment